src/components/organisms/Room/Participant.scss

```diff
@@ -0,0 +1,59 @@
+@import "scss/constants";
+
+.participant {
```

Comment:
Good opportunity for proper BEM, not just copy old obsolete practices

```suggestion
.Participant {
```

Reply:
c4879fa